docs/Concepts/Privacy/Onchain-PrivacyGroups.md

```diff
@@ -0,0 +1,47 @@
+description: Onchain Privacy Groups
+<!--- END of page meta data -->
+
+# Onchain Privacy Groups 
+
+Onchain privacy groups use smart contracts to store and maintain the group administration and distribution lists. 
+Onchain privacy groups can have their membership updated. That is, you can add and remove members when 
+using onchain privacy groups. 
+
+!!! important 
+    Onchain privacy groups are an early access feature in v1.4. Do not use in production networks. 
+    
+    The onchain privacy group interfaces may change between v1.4 and v1.5.  
+    
+    Onchain privacy groups are not compatible with [offchain privacy groups](Privacy-Groups.md). You cannot use existing 
```

Comment:
> not compatible

I don't think this is actually what you mean here. In the same logic legacy groups (an off chain group) is "not compatible" with besu orion groups (another off chain group)

Reply:
I'm going to update this to be a _we do not recommend_ style statement instead.